src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ILGeneratorImpl.cs

```diff
@@ -42,40 +41,82 @@ internal ILGeneratorImpl(MethodBuilder methodBuilder, int size)
         public override LocalBuilder DeclareLocal(Type localType, bool pinned) => throw new NotImplementedException();
         public override Label DefineLabel() => throw new NotImplementedException();
 
-        public override void Emit(OpCode opcode)
+        private static int GetStackChangeFor(StackBehaviour stackBehaviour) =>
+            stackBehaviour switch
+            {
+                StackBehaviour.Pop0
+                or StackBehaviour.Push0 => 0,
```

Comment:
I'm finding the formatting here difficult to read. Can you at least add a blank line after each line that has a => in it, in order to separate the groupings?